src/aap_eda/core/models/activation.py

```diff
@@ -102,6 +102,16 @@ class Meta:
         null=True,
         default=None,
     )
+    credentials = models.ManyToManyField(
+        "Credential", related_name="activations", default=None
+    )
+    system_credential = models.OneToOneField(
```

Comment:
@hsong-rh @Alex-Izquierdo We would need this field to encrypt the EDA_PG_NOTIFY_DSN for the pg_notify action. We cannot depend on the user creating a Vault password for this we would have to add one for the activation behind the scenes. The Activations that are using the "sources" we would be adding the EDA_PG_NOTIFY_DSN as a vaulted string.